### PR TITLE
Travis-CI: add Python versions

### PR DESCRIPTION
**Pull request by clebergnu**
This PR adds a number of jobs of different Python versions to our Travis-CI jobs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,8 @@ language: python
 python:
     - "2.7"
     - "3.4"
+    - "3.5"
+    - "3.6"
```
Comment (ldoktor) [inline .travis.yml]: iirc there is `3.7-dev` that should bring 3.7 version, right?

**Comment by clebergnu** [comment]
The discussion about using the latest requirement is a different one,
because we do need jobs in which the only thing that changes is Avocado
itself (these ones).

So, we're back again at those situations in which I propose something, and
you feel like you'd do something differently, and we reach and impasse.
Given that I sent this PR, is quite obvious that I believe in those
changes... So, let me know if besides your preference, if you see something
wrong (broken) here.

**Comment by ldoktor** [comment]
I haven't nacked it. I just asked whether this amount of testing is necessary. We don't directly pay for travis, but that doesn't mean it's for free (free things don't really exists, that's for example why I'm not on FB). So adding a "comment" meant I'd like to discuss the needs and necessity.

So would you please let me know what do you think about it? The 3.5 is in bugfix-only stage (IIRC) so having all of them seems unnecessary to me.

**Comment by ldoktor** [comment]
Yes, I know you proposed this and you weighted all the possibilities. Still I ought to ask whether you considered all risks like the travis utilization and false-negatives that keep occurring daily. Simple "I'm aware of this but I believe this matrix is the minimum set we need. About the 3.4/3.5 we plan to talk about them laters so let's test all of them until we decide." Would do for me to merge it without further questions. There is no need in reacting irritably.

Anyway let's assume you'd had said what I had written above and get this in...

**Comment by ldoktor** [comment]
only running full make-check as .travis should not affect history...

**Pull request merged**